tests/installation/welcome.pm

```diff
@@ -67,9 +67,20 @@ sub run {
     mouse_hide;
     wait_still_screen(3);
 
-    # license+lang
-    # On sle 15 license is on different screen
-    if (!sle_version_at_least('15') || is_staging()) {
+    # license+lang +product (on sle15)
+    # On sle 15 license is on different screen, here select the product
+    if (sle_version_at_least('15') && check_var('STAGING', 'Y')) {
```

Comment:
It can't get more ugly than this, right? ;-)

How temporary do you consider this depending on the specific staging project?

Reply:
as temporary as the staging project is